Apps/Src/Tasks.c

```diff
@@ -61,4 +61,11 @@ OS_SEM SendCarCAN_Sem4;
 /**
  * Global Variables
  */
-// TODO: Put all global state variables here
\ No newline at end of file
+
+const switch_states_t switchStates = {OFF, OFF, OFF, OFF, OFF, OFF, OFF, OFF, OFF, OFF, OFF};
+const blinker_states_t blinkerStates = {OFF, OFF, OFF};
+
+car_state_t carState = {0.0f, 0.0f, 0.0f, 0.0f, 0, 0, 0, switchStates, blinkerStates, OFF, OFF};
```

Comment:
I would prefer it if these were defined in the `main` function. So it'll be contained to a specific test or given case if we want to initialize things differently. Also, just to keep things in smaller scope. Keeping these global sort of circumvents the passing it to every task.